databases/insert.php

```diff
@@ -7,9 +7,9 @@
 /**
  * PDO Connection data
  */
-$dsn = 'mysql:host=localhost;dbname=ESBRegister';
+$dsn = 'mysql:host=localhost;dbname=esbregister;port=3308';
```

Comment:
i suggest to leave this file and not touch it... the dbname isn't the same and the port is usually 3304 (not 3308) but it's your machine so it's ok.
you may want to change this file accordingly...